src/auspex/instruments/holzworth.py

```diff
@@ -126,7 +126,7 @@ def ch_check(self, serial, channel):
             holzworth_driver = HolzworthPythonDriver()
             logger.debug("Using Holzworth pure-python driver.")
         except Exception as e:
-            logger.warning("Could not connect to Holzworths: {}".format(e))
+            logger.debug("Could not connect to Holzworths: {}".format(e))
```

Comment:
No warning if user gets a fake instrument?

Reply:
ill make a fix for this, it was irritating me that if someone was using a BNC this warning pops up anyway